src/translations/et.json

```diff
@@ -2,40 +2,41 @@
   "$schema": "./schema.json",
   "common": {
     "name": "Ohtlike ilmanähtuste kaart",
-    "description": "Ohtlike ilmanähtuste kaart hoiatab ilmaohtude eest.",
+    "description": "Äärmuslike ilmaolude hoiatus.",
     "not_available": "Meteoalarm pole saadaval"
   },
   "error": {
     "missing_entity": "Vajalik on Meteoalarmi olem!",
-    "missing_integration": null,
-    "invalid_integration": null
+
+    "missing_integration": "Meteoalarmi sidumine puudub!",
+    "invalid_integration": "Meteoalarmi sidumine on vigane!"    
   },
   "editor": {
-    "entity": null,
-    "integration": null,
-    "required": null,
-    "automatic": null,
-    "recommended": null,
-    "override_headline": null
+  "entity": "olem",
+  "integration": "sidumine",
+  "required": "nõutav",
+  "automatic": "vaikimisi",
+  "recommended": "soovituslik",
+  "override_headline": "pealdise alistamine"
   },
   "events": {
     "no_warnings": "Hoiatusi hetkel pole",
-    "wind": "tugev tuul",
-    "snow_ice": "lumi/jää",
-    "thunderstorms": "äike",
-    "fog": "udu",
-    "hight_temperature": "kuumalaine",
-    "low_temperature": "külmalaine",
-    "coastal_event": "rannikumeri",
-    "forest_fire": "metsapõleng",
-    "avalanches": "laviin",
-    "rain": "sademed",
-    "flood": "üleujutus",
-    "rain_flood": "paduvihm"
+    "wind": "Tugev tuul",
+    "snow_ice": "Lumi või jää",
+    "thunderstorms": "Äike",
+    "fog": "Udu",
+    "hight_temperature": "Kuumalaine",
+    "low_temperature": "Külmalaine",
+    "coastal_event": "Rannikumeri",
+    "forest_fire": "Metsapõleng",
+    "avalanches": "Laviin",
+    "rain": "Sademed",
+    "flood": "Üleujutus",
+    "rain_flood": "Paduvihm"
   },
   "messages": {
-    "yellow": "Kollane hoiatus {0}",
-    "orange": "Oranž hoiatus {0}",
-    "red": "Punane hoiatus {0}"
+    "yellow": "{0}",
+    "orange": "{0}",
+    "red": "{0}"
   }

```

Comment:
As I said, if you think that card is not properly displayed please create an issue indeed of changing translation files